java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/deploy/BuildVerifyCommand.java

```diff
@@ -241,6 +240,7 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) {
         }
     }
 
+    // TODO: ### For container-builds, should sbom generation be delegated to the task within that? If it supports it?
     private void generateBuildSbom() {
```

Comment:
@vibe13 To discuss ; Previously JBS handled SBOM after the build - I suspect now Konflux should handle that?

Reply:
At the moment we don't have any way of handling / storing the sbom - this was handled by BuildVerifyCommand but as we don't have a way in the buildah based task to access the .m2/gradle repositories I think this now needs to be handled specifically by Konflux.